server/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint consistent-return:0 */
+
+const express = require('express');
+const logger = require('./logger');
+const ngrok = require('ngrok');
+
+const frontend = require('./middlewares/frontendMiddleware');
+const isDev = process.env.NODE_ENV !== 'production';
+
+const app = express();
+
+// If you are developing more than just front-end, like POST form
+// handling or API, add your custom middleware here
+// app.use('/api', myApi);
+
+// Initialize frontend middleware that will server your JS app
+const webpackConfig = isDev
+  ? require('../internals/webpack/webpack.dev.babel')
```

Comment:
I'm not a fan of the ternary operator here, but can't come up with a nicer way to do this. Generally, ternary operators are very confusing for beginners, so if you can think of a better way...